src/plugins/lightbox/lightbox-en.hbs

```diff
@@ -210,7 +210,7 @@
 					</li>
 					<li>
 						<a href="demo/2_b.jpg" title="Image 2">
-							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" longdesc="#image2-desc" />
+							<img src="demo/2_s.jpg" alt="Image 2" aria-describedby="image2-desc" />
```

Comment:
Can you make this an exception because `longdesc` is defined by https://www.w3.org/TR/html-longdesc/

Reply:
Are you sure you want to go in that direction?

I did a bit more research this afternoon and most of the reputable sources I found downplayed the ``longdesc`` attribute.

**Pros with keeping it:**
* Certain screen readers can detect ``longdesc`` attributes, announce their presence and access them:
  * [WebAIM's ``longdest`` test cases page](https://webaim.org/techniques/alttext/longdesctestcases.htm) shows that NVDA and JAWS supported the attribute in 2015.
  * My slightly-outdated version of NVDA (2020.4) is able to detect it and optionally open a new tab that navigates to ``#image2-desc``.

**Cons with keeping it:**
* The [HTML5 Image Description Extension (longdesc) spec](https://www.w3.org/TR/html-longdesc/) became a W3C recommendation in early 2015. But [the W3C subsequently began to recognize the WHATWG HTML spec as authoritative](https://en.wikipedia.org/wiki/HTML5#W3C_and_WHATWG_conflict) 4 years later in May 2019. Wouldn't that effectively make the ``longdesc`` spec obsolete?
* Both ``htmllint`` and the [W3C's variant of the NU validator](https://validator.w3.org/nu/) treat the ``longdesc`` attribute as obsolete. Here's the specific error they produce:
  > The “longdesc” attribute on the “img” element is obsolete. Use a regular “a” element to link to the description.
* All of the images that use the ``longdesc`` attribute in the lightbox demo also use ``aria-describedby``. In fact, in my NVDA test, I got a ``longdesc`` hint upon navigating to image links... followed by the long description being automatically read out thanks to ``aria-describedby`` (both attributes refer to ``#image2-desc``).
* No one's ever suggested bringing back the ``longdesc`` attribute in [whatwg/html](https://github.com/whatwg/html).
* [PowerMapper removed its``longdesc`` tests in 2019](https://www.powermapper.com/products/sortsite/rules/w3chtml5obsoleteattributelongdesc/) after it became obsolete.
* [Deque University's long descriptions page](https://dequeuniversity.com/tips/long-descriptions) states that the ``longdesc`` attribute isn't usually recommended due to limited screen reader support. Not sure if that predates NVDA and JAWS' support, but I strongly doubt they'll change their stance about it in the future.

**Cons with removing it:**
* Screen readers that still support ``longdesc`` but that don't support ``aria-describedby`` on non-interactive elements won't be able to read the lightbox demo's long descriptions when the lightbox overlays are active. They'll only read out ``aria-describedby``'s long description on the image thumbnail links. Might not be much of a problem in practice though. Users will encounter the thumbnails links first and most screen readers probably wouldn't bother trying to view lightbox image galleries in the first place.

Reply:
Or can you update the working example that will provide an alternative for the long description.

Reply:
Upon further reflection, I think I'll fall back on your initial suggestion and keep ``longdesc`` in place for the time being with an exclusion rule.

What I have in mind for how to resolve this properly will require JS revisions. IMO that goes way beyond this PR's scope and would be better off as a spinoff PR.

Reply:
Now lightbox was put as an exception